feature/movie-detail/src/main/java/io/filmtime/feature/movie/detail/MovieDetailViewModel.kt

```diff
@@ -132,44 +133,47 @@ class MovieDetailViewModel @Inject constructor(
   }
 
   fun load() = viewModelScope.launch {
-    _state.value = _state.value.copy(isLoading = true)
-
-    when (val result = getMovieDetail(videoId)) {
-      is Success -> {
-        _state.update { state ->
-          state.copy(videoDetail = result.data, isLoading = false)
+    _state.update { it.copy(isLoading = true) }
```

Comment:
there is a known bug here that we should reset error as well. otherwise our offline implementation does not work well.

```suggestion
    _state.update { it.copy(isLoading = true, error = null) }
```